### ISSUE TITLE
Docs: Describe minimal bootable ostree system

### ISSUE THREAD

**Issue by fwilhe** [ostreedev/ostree#2964]
Hi,

as a newcomer to ostree, I feel a bit lost with the docs.

[OSTree Overview](https://ostreedev.github.io/ostree/introduction/) describes how to commit files, cool, that's easy enough.

[Deployments](https://ostreedev.github.io/ostree/deployment/) has useful info, but does not give me a full picture.

[Adapting existing mainstream distributions](https://ostreedev.github.io/ostree/adapting-existing/) is cool too but also seems to require previous knowledge that I am probably missing.

In my ideal scenario, the ostree docs would describe what a minimal ostree based bootable system is made of.

Maybe something based on [Buildroot](https://buildroot.org), or a similar minimal system that does not contain a whole actual distribution like CentOS or Fedora.

I know about meta tools such as osbuild and buildstream, but I would appreciate if there was better documentation to describe how ostree works and what its pre-requisites are.

**Comment by dharmjit** [comment]
I agree, I think such documentation will help folks who are new to `OSTree`. I felt the same when started with OSTree, Most of the docs are around OSTree constructs, and sometimes it's hard to get a grasp of the ostree based bootable OSes.

**Comment by matti** [comment]
Just read the docs and I have no idea how to actually "get started" so I did googling like "getting started ostree" with various combinations which results nothing currently.

I guess I have to move to some libostree village to get the information.

**Comment by fwilhe** [comment]
@matti What do you want to do? Is your goal to build an os based on ostree (as opposed to use it for app delivery like flatpak)?

In case you're interested in building an os, [my sample repo might be helpful](https://github.com/gardenlinux/ostree-image-builder)

**Comment by matti** [comment]
@fwilhe thanks, your sample repo is exactly what I was looking for! and the blog series seems excellent (https://blogs.sap.com/2023/07/10/making-an-immutable-image-based-operating-system-out-of-garden-linux/) reading it now.

**Comment by fwilhe** [comment]
@matti Awesome, thanks for the feedback. Feel free to reach out if you'd like to discuss that further 👍
